packages/ppf-contracts/contracts/PPF.sol

```diff
@@ -67,12 +81,12 @@ contract PPF is IFeed {
     * @param sigs Bytes array with the ordered concatenated signatures for the updates
     */
     function updateMany(address[] bases, address[] quotes, uint128[] xrts, uint64[] whens, bytes sigs) public {
-        require(bases.length != 0);
-        require(bases.length == quotes.length);
-        require(bases.length == xrts.length);
-        require(bases.length == whens.length);
-        require(bases.length == sigs.length / 65);
-        require(sigs.length % 65 == 0);
+        require(bases.length != 0, ERROR_BAD_BASE_ADDRESSES_LENGTH);
+        require(bases.length == quotes.length, ERROR_BAD_QUOTE_ADDRESSES_LENGTH);
```

Comment:
Could we rename these errors so they better represent the intents:

1. zero length
1. length mismatch